test/performance/vectorization/vectorPragmas/zipIterInFollowerNoVectorForeach.chpl

```diff
@@ -0,0 +1,40 @@
+module iters {
+
+  iter nonInlinableIter(r: range) {
+    foreach i in r # (r.size - 1) do 
+      yield i;
+    yield r.last;
+  }
+
+  iter myiter(nn: int, nt: int) {
+    foreach i in 0..#nt {
+      foreach j in i*nn..#nn {
```

Comment:
it seems to me that this test would be better if the serial version is also not inlineable